lib/Model/IMessage.php

```diff
@@ -131,6 +131,14 @@ public function getAttachments(): array;
 	 */
 	public function addRawAttachment(string $name, string $content): void;
 
+	/**
+	 * @param string $name
+	 * @param string $content
```

Comment:
might be worth noting the expected rfc822 format for the string

Reply:
Good idea